state/protocol/badger/mutator_test.go

```diff
@@ -470,6 +470,20 @@ func TestExtendEpochTransitionValid(t *testing.T) {
 		metrics.On("CurrentEpochPhase", initialPhase).Once()
 		metrics.On("CommittedEpochFinalView", finalView).Once()
 
+		metrics.On("CurrentEpochFinalView", finalView).Once()
+
+		currentDKGPhase1FinalView, err := initialCurrentEpoch.DKGPhase1FinalView()
+		require.NoError(t, err)
+		metrics.On("CurrentDKGPhase1FinalView", currentDKGPhase1FinalView).Once()
+
+		currentDKGPhase2FinalView, err := initialCurrentEpoch.DKGPhase2FinalView()
+		require.NoError(t, err)
+		metrics.On("CurrentDKGPhase2FinalView", currentDKGPhase2FinalView).Once()
+
+		currentDKGPhase3FinalView, err := initialCurrentEpoch.DKGPhase3FinalView()
+		require.NoError(t, err)
+		metrics.On("CurrentDKGPhase3FinalView", currentDKGPhase3FinalView).Once()
```

Comment:
Nice 👍 

Could we also assert that the metrics are updated correctly after the epoch transition. Can use the same pattern as is used for checking current epoch counter on lines 697/702 below:
https://github.com/onflow/flow-go/blob/a603c086d66b5e51e73c39d6ab6485f70d710cd8/state/protocol/badger/mutator_test.go#L695-L705